fontir/src/ir.rs

```diff
@@ -1035,19 +1035,31 @@ pub enum AnchorKind {
     /// An attachment anchor on a mark glyph
     Mark(GroupName),
     /// A base attachment on a ligature glyph
-    Ligature { group_name: GroupName, index: usize },
+    Ligature {
+        group_name: GroupName,
+        index: usize,
+    },
     /// An anchor marking the presence of a ligature component with no anchors.
     ///
     /// These are names like '_3'.
     ComponentMarker(usize),
-    // we will add more anchor kinds in the future like entry/exit
+    CursiveEntry,
```

Comment:
is it possible to have multiple entry/exist anchors? Fine if we don't handle that on a first pass, but is it something we need to consider?